src/iperf_vsock.c

```diff
@@ -98,13 +98,23 @@ static struct sockaddr *
 vsock_sockaddr(const char *cid_str, int port, int listen, socklen_t *len)
 {
 	char *end = NULL;
+	int sibling = 0;
 	long cid;
 
+
 	if (cid_str == NULL) {
 		return NULL;
 	}
 
-	cid = strtol(cid_str, &end, 10);
+	/*
+	 * Connection to a sibling VM
+	 * cid_str is of the form "s<cid>"
+	*/
+	if (cid_str[0] == 's') {
+		sibling = 1;
```

Comment:
what about doing here also `cid_str++`, so we can use `cid = strtol(cid_str, &end, 10);` in both cases

Reply:
no, we will break the path for the unix domain socket, so better how you did ;-)